backends/apple/coreml/CMakeLists.txt

```diff
@@ -111,32 +118,50 @@ set(PROTOBUF_SOURCES
     runtime/sdk/format/WordTagger.pb.cc
 )
 
+find_library(FOUNDATION_FRAMEWORK Foundation)
+
+# CoreML util
+add_library(coreml_util)
+target_sources(coreml_util PRIVATE ${UTIL_SOURCES})
```

Comment:
nit: why not put the sources in add_library directly?

Reply:
I didn't know that was possible!